easybuild/easyconfigs/t/Trinity/Trinity-2.15.2-foss-2023a.eb

```diff
@@ -0,0 +1,64 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia
+# Homepage:     https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::     Robert Qiao <rob.qiao@flinders.edu.au>
+# License::     Custom
+#
+# Notes::
+##
+
+name = 'Trinity'
+version = '2.15.2'
+
+homepage = 'https://trinityrnaseq.github.io'
+description = """Trinity represents a novel method for the efficient and robust de novo reconstruction
+ of transcriptomes from RNA-Seq data. Trinity combines three independent software modules: Inchworm,
+ Chrysalis, and Butterfly, applied sequentially to process large volumes of RNA-Seq reads."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://github.com/trinityrnaseq/trinityrnaseq/releases/download/%(name)s-v%(version)s']
+sources = ['trinityrnaseq-v%(version)s.FULL.tar.gz']
+patches = ['Trinity-%(version)s_fix-bamsifter.patch']
+checksums = [
+    {'trinityrnaseq-v2.15.2.FULL.tar.gz': 'baab87e4878ad097e265c46de121414629bf88fa9342022baae5cac12432a15c'},
+    {'Trinity-2.15.2_fix-bamsifter.patch': 'f557a3d462218e27f3601ac07edd2bbafe5fdb088ab81f642e7025edfe3e48ef'},
+]
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('CMake', '3.26.3'),
+]
+
+# for reference, list of dependencies in the container image used upstream:
+# https://github.com/trinityrnaseq/trinityrnaseq/blob/master/Docker/Dockerfile
+dependencies = [
```

Comment:
DB_File.pm is missing. Need to add DB_File as dep `('DB_File', '1.859'),`

Reply:
I think it's ('DB', '18.1.40') that is missing DB was a dependency of  Perl-5.34.1-GCCcore-11.3.0.eb but is not a part of 
```
 ('Perl', '5.36.1'),
 ('Perl-bundle-CPAN', '5.36.1'),
```